l10n_ch_fds_upload_sepa/wizards/fds_files_import_tobankstatments_wizard.py

```diff
@@ -0,0 +1,34 @@
+from odoo import models, api
+
+import base64
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class FdsFilesImportToBankStatementsWizardPain000(models.TransientModel):
+    _inherit = 'fds.files.import.tobankstatments.wizard'
+
+    @api.multi
+    def _import2bankStatements(self, fds_files_ids):
+        account_pain000 = self.env['account.pain000.parser']
+        for fds_file in fds_files_ids:
+            for pf_file in fds_file:
```

Comment:
Why do you have two loops? fds_files_ids is a recordset, only one loop is required as you cannot loop over a single record in the recordset.